README.md

```diff
@@ -50,10 +52,47 @@ In addition to `live_view_native` you may want to include some additional librar
 {:live_view_native_live_form, "~> 0.3.0-rc.3"}
 ```
 
+The versions above may be subject to change, so make sure to check the library documentation for the latest version.
```

Comment:
I don't think this block is necessary, we'll be updating this section of the README along with releases and the `~>` allows for relaxed version constraints so it should install newer versions within the semver range